Assignment4/pablohdez98-21A289/task08.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+"""Task08.ipynb
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/16uRIkVihZRhtS9VAQEjICHoOm1hkhSsH
+
+**Task 08: Completing missing data**
+"""
+
+!pip install rdflib
+github_storage = "https://raw.githubusercontent.com/FacultadInformatica-LinkedData/Curso2021-2022/master/Assignment4/course_materials"
+
+from rdflib import Graph, Namespace, Literal, URIRef
+g1 = Graph()
+g2 = Graph()
+g1.parse(github_storage+"/rdf/data01.rdf", format="xml")
+g2.parse(github_storage+"/rdf/data02.rdf", format="xml")
+
+"""Tarea: lista todos los elementos de la clase Person en el primer grafo (data01.rdf) y completa los campos (given name, family name y email) que puedan faltar con los datos del segundo grafo (data02.rdf). Puedes usar consultas SPARQL o iterar el grafo, o ambas cosas."""
+
+from rdflib.namespace import RDF, RDFS
+
+NS = Namespace("http://data.org#")
+VCARD = Namespace("http://www.w3.org/2001/vcard-rdf/3.0#")
+
+for s1, p1, o1 in g1.triples((None, RDF.type, NS.Person)):
+  for s2, p2, o2 in g2.triples((s1, None, None)):
+    g1.add((s1, p2, o2))
```

Comment:
You are not restricting yourself to the requested fields. You are completing all fields.